coralproject/talk
# Banned users should not be able to report/flag anything from the Front End

## What does this PR do?

- Allow to click on 'Report' only if the user is not banned.

## How do I test this PR?

- login with a banned user
- try to report on a comment or username
- it should not click

Issue #191
